cypress/fixtures/constants.ts

```diff
@@ -283,6 +283,12 @@ export class Constants {
             host: 'Host button: ',
             remote: 'Remote button: '
         },
+
+        reactHostRemoteContainerHeader: 'This is the container App hosted at localhost:8080',
+        reactHostRemoteContainerDiv: 'This component is from the Host React App hosted at localhost:8081',
+        reactHostRemoteHostedDiv: 'This is the Remote App hosted at localhost:8081)',
+        reactHostRemoteHostedInvoices: 'Invoices',
```

Comment:
merge Invoices and Expenses consts into one object with named variables, as you can see in example ended on 285 string

Reply:
fixed